### PR TITLE
Mention that in Fedora you can now install scancode from Fedora repo

### LINKED ISSUE #3095: Build RPMs for scancode
The LSB and ISO Linux was the genisis of the Linux Foundation.   The scancode-toolkit needs to live up to the LF standards of portability.   Either [fix to build](https://github.com/nexB/scancode-toolkit/issues/3094) or offer noarch RPMs.

### PR DESCRIPTION
**Pull request by xsuchy**
See https://bodhi.fedoraproject.org/updates/?packages=scancode-toolkit

Fixes: #3095

### Tasks

* [x] Reviewed [contribution guidelines](https://github.com/nexB/scancode-toolkit/blob/develop/CONTRIBUTING.rst)
* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] Tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
  Run [tests](https://scancode-toolkit.readthedocs.io/en/latest/contribute/contrib_dev.html#running-tests) locally to check for errors. 
* [x] Commits are in uniquely-named feature branch and has no merge conflicts 📁
* [x] Updated documentation pages (if applicable)
* [x] Updated CHANGELOG.rst (if applicable)

### CODE REVIEW

**Comment by AyanSinhaMahapatra** [comment]
Thanks a lot @xsuchy!
Could you also mention this in the installation docs at https://github.com/nexB/scancode-toolkit/blob/develop/docs/source/getting-started/install.rst?plain=1 so this shows up at https://scancode-toolkit.readthedocs.io/en/latest/getting-started/install.html

This would be a new section and reference here at https://github.com/nexB/scancode-toolkit/blob/develop/docs/source/getting-started/install.rst?plain=1#L414 and a reference at https://github.com/nexB/scancode-toolkit/blob/develop/docs/source/getting-started/install.rst?plain=1#L36, both like the `pip install` sections above.

**Comment by pombredanne** [comment]
@xsuchy you rock :rocket:

**Comment by xsuchy** [comment]
> @xsuchy you rock 🚀

Nah. I did just a fraction of the work. Credits go to Eclipseo.

**Comment by xsuchy** [comment]
> Thanks a lot @xsuchy! Could you also mention this in the installation docs 

Updated.

File: docs/source/getting-started/install.rst [hunk 1]
```diff
@@ -412,6 +417,22 @@ To uninstall, run::
...
 
 
+----
+
+.. _fedora_install:
+
+Install from Fedora's repository
+--------------------------------
+
+The package is available in Fedora 40 and newer. Run:
```
Comment (AyanSinhaMahapatra) [inline docs/source/getting-started/install.rst]:
```suggestion
The package is available in Fedora 40 and newer. Run::
```

File: docs/source/getting-started/install.rst [hunk 2]
```diff
@@ -412,6 +417,22 @@ To uninstall, run::
...
+
+    dnf install scancode-toolkit
+
+To uninstall, run:
```
Comment (AyanSinhaMahapatra) [inline docs/source/getting-started/install.rst]:
```suggestion
To uninstall, run::
```

**Review by AyanSinhaMahapatra — APPROVED** [review]
@xsuchy Thanks!
We need just these tiny changes to highlight these correctly in the docs, and this is ready to merge otherwise.

**Comment by AyanSinhaMahapatra** [comment]
Thanks! Merging this.

**Pull request merged**